velox/docs/monthly-updates/january-2023.rst

```diff
@@ -0,0 +1,97 @@
+********************
+January 2023 Update
+********************
+
+Core Library
+============
+
+* Support for Generic writer.
+
+
+Memory Pool
+=============
+
+* Memory code refactor.
+* Fix memory pool to always return valid buffer.
```

Comment:
@xiaoxmeng Meng, would you help phrase these in ways that can be understood by library users?